dav1d-sys/Cargo.toml

```diff
@@ -9,11 +9,18 @@ edition = "2021"
 
 build = "build.rs"
 
-[package.metadata.system-deps]
-dav1d = "1.0.0"
-
 [build-dependencies]
 system-deps = "6.0"
 
 [dependencies]
 libc = "0.2"
+
+[features]
+v1_1 = []
```

Comment:
I'd make it default now.

Reply:
That defeats the purpose of making it optional IMHO :)

Reply:
You are right.